tests/gateway-tests/test/testkit/subscriptions_client.go

```diff
@@ -50,8 +50,6 @@ func (sc *client) Create(namespace, application, eventType string) error {
 		SubscriptionSpec: v1alpha1.SubscriptionSpec{
 			Endpoint:                      "https://some.test.endpoint",
 			IncludeSubscriptionNameHeader: true,
-			MaxInflight:                   400,
```

Comment:
IMO, we need both versions of the test as the old clients can come with these fields(MaxInflight...).

Reply:
Here the v1alpha1.SubscriptionSpec does not have these properties anymore, so we have to remove them.